pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -183,6 +184,7 @@ private void filterOsteoFromOtherOsteoOutForExport(UnparsedESParticipantDto unpa
             filterData((Map<String, List<Map<String, Object>>>) dsmObject, ESObjectConstants.KIT_REQUEST_SHIPPING, ddpInstanceId);
             filterData((Map<String, List<Map<String, Object>>>) dsmObject, ESObjectConstants.MEDICAL_RECORD, ddpInstanceId);
             filterData((Map<String, List<Map<String, Object>>>) dsmObject, ESObjectConstants.ONC_HISTORY_DETAIL, ddpInstanceId);
+            filterData((Map<String, List<Map<String, Object>>>) dsmObject, ESObjectConstants.SOMATIC_RESULT_UPLOAD, ddpInstanceId);
```

Comment:
We can take this offline,  but I am also curious about what this file does.

Reply:
Routes use this "wrapper" to build a collection of data from the participants_structured index.  Happy to discuss more offline.  IMHO, this needs more testing and is the source of some of the OS1/OS2 issues.